adoptingerlang.org

```diff
@@ -2860,10 +2858,14 @@ Use of =COPY . .= is discouraged because it makes it more likely to invalidate t
 
 Copying files into an image and caching layers are no longer the only options for efficiency when building a Docker image. Caching the built dependencies in a layer is all well and good but that layer also contains the Hex package cache Rebar3 creates under =~/.cache/rebar3/hex=. Any change to =rebar.config= or =rebar.lock= will result in all the packages having to not only be rebuilt but also be re-fetched from Hex. Additionally, the instruction to copy in the whole project, creating an additional layer with all the source is wasteful since we only care about the build artifacts.
 
-These issues are resolved as of Docker 19.03 through an experimental syntax for mounting files to the context of a =RUN= command. To enable the experimental syntax the environment variable =DOCKER_BUILDKIT= must be set or ={"features":{"buildkit": true}}= be set in =/etc/docker/daemon.json=, and =# syntax = docker/dockerfile:experimental= used as the first line of the Dockerfile:
+These issues are resolved as of Docker 19.03 through an experimental syntax for
+mounting files to the context of a =RUN= command. To enable the experimental
+syntax the environment variable =DOCKER_BUILDKIT= must be set or
+={"features":{"buildkit": true}}= be set in =/etc/docker/daemon.json=, and =#
+syntax=docker/dockerfile:1.2= used as the first line of the Dockerfile:
```

Comment:
is splitting the markup really working here where the `=#` is on one line and the rest on another one?

Reply:
Yup, I just checked the preview deploy https://651400cdecfab7009b23a4fb--inspiring-shannon-000ae0.netlify.app/docs/production/docker/